docs/pages/reference/cli/tbot.mdx

```diff
@@ -17,6 +17,7 @@ The primary commands for `tbot` are as follows:
 | `tbot init` | Initialize a certificate destination directory for writes from a separate bot user, configuring either file or POSIX ACL permissions. |
 | `tbot db` | Connects to databases using native clients and queries database information. Functions as a wrapper for `tsh`, and requires `tsh` installation. |
 | `tbot proxy` | Allows for access to Teleport resources on a cluster using TLS Routing. Functions as a wrapper for `tsh`, and requires `tsh` installation. |
+| `tbot tpm identify` | Output identifying information related to the TPM detected on the system. |
```

Comment:
I would expand "TPM" here since this is the first mention

Reply:
91e8b39972